src/utils/hashPassword.js

```diff
@@ -0,0 +1,13 @@
+const bcrypt = require('bcryptjs');
+
+const hashPassword = (password, callback) => {
```

Comment:
try to find if bcrypt support promise, So use promise instead of callback, either convert it to promise